lib/generators/pageflow/theme/theme_generator.rb

```diff
@@ -13,9 +13,10 @@ def copy_template
         directory('images/pageflow/themes/default', themes_path('images', name))
 
         template('stylesheets/pageflow/themes/default.css.scss', themes_path('stylesheets', "#{name}.css.scss")) do |content|
-          content.gsub!('$theme-name: "default";', %Q'$theme-name: "#{name}";')
           content.gsub!('@import "./default/', %Q'@import "./#{name}/')
         end
+
+        gsub_file themes_path('stylesheets', "#{name}/variables.css.scss"), /default/, name
```

Comment:
I had a feeling you would say that. :grin: 

Agreed. It makes it more brittle in case anyone changes the double quotes to singles. But it becomes less dangerous too. I'll make the change.